website_product_brand/oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+product-attribute
```

Comment:
This file is put at root level, not here, and it's not a link, but a file with contents